### PR TITLE
58 fix header styling kk

### PR DESCRIPTION
**Pull request by komalgill0310**
### **What changes did I make? **
1. I changed the html elements structure for the Header. 
2. I modified the CSS for the Header. 

### **Why did I make these changes? **
1. For a Larger screen, the header image shows the full image without hiding some of the image's content when taking the entire width of the screen. 

### **Visual Change**
recipeHeaderImg

### **Known Issue**
1. As the screen size gets bigger, the image stretches. 

### **Seeking Help**
I would be grateful for assistance in preventing image stretching. Thank you for your help!

### CODE REVIEW

**Review by AOA19 — CHANGES_REQUESTED** [review]
@komalgill0310, 
- The issue with the header image was the fact that the image would be be too zoomed in on larger screens. And I see you tried to resolve that by placing the image inside an `` tag and adjusting the width and height. But as you pointed out the image now stretches as you size to larger screens.
- So my suggestion would be, keep the image as a background image like it was before, that way you don't have to `position: absolute;` the two header tags.
- Take the images for each screen size (mobile, tablet, desktop and desktop large) from the Figma design, add them to our public folder. 
- Set up media queries for each screen size and add the image for that specific screen size inside the `background-image` or `background` property. This way when the screen is resized the image will look similar to the Figma design 
- I would also adjust the `max-height` value depending on the screen size.

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -3,13 +3,24 @@ import './Header.css';
...
-      <h1 className="font-semibold">What will you make today?</h1>
-      <h2 className="">
-        App name is an app that helps you find and create delicious dishes base on your ingredients!
-      </h2>
+    <div className="header">
+      <img src="/header_img.jpg" alt="" />
+      <div className="text-center place-items-center">
+        <h1 className="font-semibold">What will you make today?</h1>
+        <h2 className="">
+          App name is an app that helps you find and create delicious dishes base on your ingredients!
```
Comment (AOA19) [inline src/components/header/Header.jsx]: @komalgill0310  Also can you change "App name" to FlavrFindr

Comment (komalgill0310) [reply src/components/header/Header.jsx]: @AOA19 Thank you for your review! I have pushed the updated changes. Please review.

**Review by AOA19 — APPROVED** [review]
@komalgill0310, The header looks good, I did make a a few small changes. I noticed that the logo was stretching a bit so I replaced the png file with an svg, and I removed the comments that were inside of the Header component. I'll go ahead and merge these changes

**Pull request merged**